docs/faq.md

```diff
@@ -0,0 +1,91 @@
+# Frequently Asked Questions
+
+## I'm about to document a new PHP extension. How should I start?
+Change your working directory to `phpdoc/doc-base/scripts/docgen/` and execute:
+```
+php docgen.php -e simplexml -o outdir
+```
+
+It creates the skeletons that you edit, and then commit.
+
+Help is available with following command: `php docgen.php -h`.
+
+## I created skeletons that contain a bunch of default text, should I commit it?
+No! Edit the files, to check the generated content and add more information,
+before committing. Thinking that it is okay to commit the skeleton files because
+you will soon come along and flesh them out might seem like a good idea. However,
+temporary often becomes permanent.
+
+## Running configure.php ends up segfaulting, what is up?
+There are bugs with certain versions of libxml that cause this, so hacks exist to get around it.
+To execute the hack, pass in: `$ php configure.php --disable-segfault-error`.  
+Note: This disables some error checking and beautification but raw errors will be shown.  
+Note: Usually the problem is a major XML syntax issue.
```

Comment:
This was a PHP 5 thing, I don't think this flag even exists anymore in configure.